doc/install_centos6.txt

```diff
@@ -16,44 +16,16 @@ Easy Installation of an optimized Theano on CentOS 6
     investing money (i.e. buying a license to a proprietary BLAS
     implementation), it is possible to gain further performance.
```

Comment:
Aren't you confusing Enthought and Conda ? I think that contrary to Enthought, Conda is always free no ?

But ok I totally agree with the rest of what you said.